internal/generator/vector/output/otlp/otlp_all.toml

```diff
@@ -26,6 +26,16 @@ source = '''
       {"key": "k8s.container.name", "value": {"stringValue": get!(.,["kubernetes","container_name"])}},
       {"key": "k8s.namespace.name", "value": {"stringValue": get!(.,["kubernetes","namespace_name"])}}]
   )
+  # Append backward compatibility attributes
+  resource.attributes = append( resource.attributes,
+      [{"key": "log_type", "value": {"stringValue": .log_type}}]
```

Comment:
I'm working in this area as well.  I think this may break the current tests because of grouping of some of the infra logs.  I believe we may need to couple this change with "log_source" and probably cluster node name....